src/main/groovy/com/ullink/gradle/plugins/jibx/JIBXPlugin.groovy

```diff
@@ -11,33 +11,40 @@ class JIBXPlugin implements Plugin<Project> {
         project.configurations.create('jibxCompile')
         project.extensions.create('jibx', JIBXPluginExtension)
 
+        def output = project.sourceSets.main.output
+        def mainClassesDirs
```

Comment:
Could be
`mainClassesDirs = output.hasProperty('classesDirs') ? output.classesDirs : output.classesDir`
?